Fix eslint issues

src/components/SurveyForm.js
   4:8   error  "mapProps" is defined but never used                    no-unused-vars
  50:7   error  'active' is missing in props validation for SurveyForm  react/prop-types
  85:90  error  Missing space before value for key "marginLeft"         key-spacing

src/components/WidgetForm.js
  48:25  error  color is already declared in the upper scope  no-shadow

src/redux/clientMiddleware.js
  7:1  error  Trailing spaces not allowed  no-trailing-spaces

src/redux/create.js
  1:23  error  "combineReducers" is defined but never used  no-unused-vars

src/views/Survey.js
  64:5  warning  Unexpected alert  no-alert

✖ 7 problems (6 errors, 1 warning)

--- a/.eslintrc
+++ b/.eslintrc
@@ -15,7 +15,8 @@
     "padded-blocks": 0,
     "comma-dangle": 0,  // not sure why airbnb turned this on. gross!
     "indent": [2, 2, {"SwitchCase": 1}],
-    "no-console": 0
+    "no-console": 0,
+    "no-alert": 0
   },
   "plugins": [
     "react"

--- a/src/components/SurveyForm.js
+++ b/src/components/SurveyForm.js
@@ -1,7 +1,6 @@
 import React, {Component, PropTypes} from 'react';
 import {connectReduxForm} from 'redux-form';
 import surveyValidation from '../validation/surveyValidation';
-import mapProps from 'map-props';
 
 function asyncValidate(data) {
   // TODO: figure out a way to move this to the server. need an instance of ApiClient
@@ -30,6 +29,7 @@ function asyncValidate(data) {
 export default
 class SurveyForm extends Component {
   static propTypes = {
+    active: PropTypes.bool.isRequired,
     asyncValidating: PropTypes.bool.isRequired,
     fields: PropTypes.object.isRequired,
     dirty: PropTypes.bool.isRequired,
@@ -82,7 +82,7 @@ class SurveyForm extends Component {
               <button className="btn btn-success" onClick={handleSubmit}>
                 <i className="fa fa-paper-plane"/> Submit
               </button>
-              <button className="btn btn-warning" onClick={resetForm} style={{marginLeft:15}}>
+              <button className="btn btn-warning" onClick={resetForm} style={{marginLeft: 15}}>
                 <i className="fa fa-undo"/> Reset
               </button>
             </div>

--- a/src/components/WidgetForm.js
+++ b/src/components/WidgetForm.js
@@ -45,7 +45,7 @@ export default class WidgetForm extends Component {
                   value={color.value}
                   onChange={handleChange('color')}
                   onBlur={handleBlur('color')}>
-            {colors.map(color => <option value={color} key={color}>{color}</option>)}
+            {colors.map(valueColor => <option value={valueColor} key={valueColor}>{valueColor}</option>)}
           </select>
           {color.error && color.touched && <div className="text-danger">{color.error}</div>}
         </td>

--- a/src/redux/clientMiddleware.js
+++ b/src/redux/clientMiddleware.js
@@ -4,7 +4,7 @@ export default function clientMiddleware(client) {
       if (typeof action === 'function') {
         return action(dispatch, getState);
       }
-      
+
       const { promise, types, ...rest } = action;
       if (!promise) {
         return next(action);

--- a/src/redux/create.js
+++ b/src/redux/create.js
@@ -1,4 +1,4 @@
-import { createStore, combineReducers, applyMiddleware, compose } from 'redux';
+import { createStore, applyMiddleware, compose } from 'redux';
 import createMiddleware from './clientMiddleware';
 
 export default function createApiClientStore(client, data) {

